### PR TITLE
HOCS-2310: change cron-job

### PR DESCRIPTION
**Pull request by LiamMacP**
We have currently just changed the implementation of the cronjob. 
This is now set to run early morning (within system uptime varying by 
system). This change also removes the unnecessary 
role header and changes the curl to accommodate the new endpoint 
url. We have also added a custom user agent header to the curl 
to allow for better identification in the logs.

### CODE REVIEW

File: kd/refreshDcuAggregatedCasesView.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: CronJob
 metadata:
   name: hocs-refresh-dcu-caseview
 spec:
-  schedule: "0 22 * * 1-5"
+  schedule: "15 {{.REFRESH_CASE_START_HOUR}}-{{.REFRESH_CASE_END_HOUR}} * * 1-5"
```
Comment (zuzak) [inline kd/refreshDcuAggregatedCasesView.yaml:6]: I'd just have a `$REFRESH_CRON` string tbh

**Review by agraham19 — APPROVED** [review]

File: kd/refreshDcuAggregatedCasesView.yaml [hunk 2]
```diff
@@ -29,7 +29,6 @@ spec:
               args:
                 - >
                   http_status=$( curl -vk -X POST -u ${HOCS_BASICAUTH} -w '%{http_code}'
```
Comment (zuzak) [inline kd/refreshDcuAggregatedCasesView.yaml:31]: Might be worth setting a unique User-Agent header here, just to make logsearching easier

**Review by zuzak — APPROVED** [review]

File: kd/refreshDcuAggregatedCasesView.yaml (hunk 1 above)
Comment (LiamMacP) [reply kd/refreshDcuAggregatedCasesView.yaml:6]: Yeah I prefer that! Will change it up to onlly run at specific time every day depending on system.

File: kd/refreshDcuAggregatedCasesView.yaml (hunk 2 above)
Comment (LiamMacP) [reply kd/refreshDcuAggregatedCasesView.yaml:31]: Do we actually store that in Kibana?

File: kd/refreshDcuAggregatedCasesView.yaml (hunk 1 above)
Comment (LiamMacP) [reply kd/refreshDcuAggregatedCasesView.yaml:6]: Done

File: kd/refreshDcuAggregatedCasesView.yaml (hunk 2 above)
Comment (LiamMacP) [reply kd/refreshDcuAggregatedCasesView.yaml:31]: Done

**Pull request merged**